jenkins/opensearch/perf-test.jenkinsfile

```diff
@@ -177,19 +177,23 @@ pipeline {
                             postCleanup()
                         }
                         aborted {
-                            script {
-                                def stackName = "test-single-security-${BUILD_ID}-${ARCHITECTURE}-perf-test"
+                             script {
+                                stackName = "test-single-security-${BUILD_ID}-${ARCHITECTURE}-${BUILD_NUMBER}"
```

Comment:
instead of 2 `aborted` runs can we have a list of stack names, sec and non-sec, and just iterate over each name and delete? @jordarlu

Reply:
thanks for the reviewing and commenting, @rishabh6788 .. yep.. let me move the code block to the outter post - aborted section, and I can loop thru both secuiry and non-security stacks.. It will be celaner this way.  Thanks!!